python/pyspark/mllib/feature.py

```diff
@@ -447,6 +454,17 @@ class Word2Vec(object):
     >>> syms = model.findSynonyms(vec, 2)
     >>> [s[0] for s in syms]
     [u'b', u'c']
+
+    >>> import os, tempfile
+    >>> path = tempfile.mkdtemp()
+    >>> model.save(sc, path)
+    >>> sameModel = Word2VecModel.load(sc, path)
+    >>> model.transform("a") == sameModel.transform("a")
+    True
+    >>> try:
+    ...     os.removedirs(path)
```

Comment:
> It seems that the official Jenkins works without errors.
Yes, but I meant that without the try, except clause. The directory will never be removed since an OSError will always raise, since it is not empty.